### PR TITLE
Michael  cynthia/#62/app registration pages

### PR DESCRIPTION
**Pull request by chelahcynthia**
This PR registers new users.

### CODE REVIEW

**Comment by chelahcynthia** [comment]
Managed to fix the lint errors. I'll be waiting for the review

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,28 @@
 import Home from "./pages/Home";
+import Registration from "./pages/Registration";
 import {Route, Routes} from "react-router-dom";
+import { ApolloClient, InMemoryCache, ApolloProvider} from "@apollo/client";
+
+
+
+// const link = from([
+//   new HttpLink({ uri: "http://localhost:3000/graphql" }),
+// ]);
```
Comment (otsembo) [inline client/src/App.tsx]: Remove unnecessary comment from code

File: client/src/pages/Registration.tsx [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+      </div>
+      </form>
+      <p className=' text-center  mx-auto w-[305px] '>
+        By continuing you accept our standard
+        <span className='underline px-2'>terms and conditions</span>
+        and our <span className=' px-2 underline'>privacy policy.</span>
+      </p>
+      <p className='text-center flex justify-center pb-12 '>
+        Already have an account?
+        <p>Sign In</p>
```
Comment (otsembo) [inline client/src/pages/Registration.tsx]: Add some whitespace between `Already have an account?` and `Sign In`.

**Review by otsembo — CHANGES_REQUESTED** [review]
Hello @chelahcynthia @MICHAELMUNAVU83 Good job on the PR.
I have requested minor changes to the PR.

I will add my code to the PR by tomorrow, Tuesday 6 June EOD.

File: .DS_Store
Comment (otsembo) [inline .DS_Store:1]: Remove this file from version control.

**Review by otsembo — CHANGES_REQUESTED** [review]
Hello @chelahcynthia @MICHAELMUNAVU83 Great job on this PR.
Apologies for not sending in this review earlier, I was facing some small blocker with the codebase but that is now fixed.

# FIXES
Check on the following:

- [ ] The Add photo functionality does not seem to be working correctly.
- [ ] A user is able to swipe right even if the register process fails. This should not be the case. Find a way to prevent this from happening.
- [ ] The `reason for joining` section also does not seem to work. Kindly check on this as well.
- [ ] Once the reason for joining is clicked, it should be highlighted so that the user can be able to know which button they have clicked.

# ENHANCEMENTS
The screen does not scale well in larger screens. Kindly find a way to sort this out. You can acquire inspiration from what the guys working on `login` did.
image

**NB: Make sure you pull the changes made, since I have added quite a large number of them**

I have also added a few more lint checkers and code style actions. I have also updated the `README` with some new instructions on how to use them.

cc @timothykiyeng => You can help out with the upload section for images here.

**Review by otsembo — CHANGES_REQUESTED** [review]
Hello @MICHAELMUNAVU83 
You seem to be having a failing check on the PR.
Have a look at that.

**Comment by MICHAELMUNAVU83** [comment]
Hello @otsembo  , I have fixed the error present .

**Review by otsembo — CHANGES_REQUESTED** [review]
Hello @MICHAELMUNAVU83 
There seems to be an issue with how the app navigation works when creating an account.
[Video](https://github.com/Amaris-Digital/fariji/assets/38042565/b0531535-5045-4283-97e1-390354e70031)

# FIXES

- [x] The `Exit` action returns the page to previous slide as opposed to resetting to the initial page.
- [x] Once an error occurs during registration, the page still navigates to the next slide regardless, this should not be the case. Ensure you confirm that the error is non-existent before moving to the next page.

**Comment by MICHAELMUNAVU83** [comment]
Hello @otsembo  , I have been able to do the exit feature but I have struggled with the other one , any help accorded would be greatly appreciated .

**Review by otsembo — APPROVED** [review]
Everything looks good.

**Pull request merged**